### PR TITLE
Fix testMultipleMembershipsContribution to use order api

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
This test was not creating valid financial records prior to this

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
@colemanw can I get a merge on this?

**Comment by eileenmcnaughton** [comment]
@colemanw @seamuslee001 @demeritcowboy  this should be a straight forward merge - unlike the stalled one it does not have a date change in it

File: tests/phpunit/api/v3/MembershipTest.php [hunk 1]
```diff
@@ -189,97 +189,86 @@ public function testActivityForCancelledContribution(): void {
...
+    $memberships = $this->callAPISuccess('Membership', 'get', ['sort' => 'contact_id', 'sequential' => 1])['values'];
 
-    // check for Membership 1
-    $params = ['id' => $membershipID1, 'return' => ['contact_id', 'status_id']];
-    $membership1 = $this->callAPISuccess('Membership', 'get', $params);
-    $result1 = $membership1['values'][$membershipID1];
-    $this->assertEquals($result1['contact_id'], $contactId1);
-    $this->assertEquals($result1['status_id'], $newMembershipId);
+    $this->assertEquals($contactId1, $memberships[0]['contact_id']);
+    $this->assertEquals('test status', CRM_Core_PseudoConstant::getName('CRM_Member_BAO_Membership', 'status_id', $memberships[0]['status_id'], 'New'));
```
Comment (demeritcowboy) [inline tests/phpunit/api/v3/MembershipTest.php:263]: I think this PR is ok but the `'New'` was confusing me, but it seems getName() only takes 3 params so the New does nothing. Was it meant to check that the membership is in a New state?

Comment (colemanw) [reply tests/phpunit/api/v3/MembershipTest.php:263]: Good catch. Can you fix this @eileenmcnaughton and then it can be MOP.

Comment (eileenmcnaughton) [reply tests/phpunit/api/v3/MembershipTest.php]: @colemanw @demeritcowboy fixed - thanks!

**Pull request merged**